components/ProjectCard.js

```diff
@@ -20,7 +23,8 @@ const ProjectList = styled.ul`
   position: relative;
   flex-direction: column;
   align-items: start;
-  padding-left: 13px;
+  padding-left: 15px;
```

Comment:
For padding and margin its better to use relative units like `em` or `rem`, so that they change depending on the font size. 
What really helped me using these units was remembering that 16px is about 1rem. So your padding-left could be 0.9 or 1rem